app/src/main/java/com/duckduckgo/app/email/EmailManager.kt

```diff
@@ -100,6 +103,19 @@ class AppEmailManager(
         }
     }
 
+    data class UserData(val token: String?, val userName: String?, val nextAlias: String?)
+    var moshi = Builder().build()
+    var jsonAdapter: JsonAdapter<UserData> = moshi.adapter(UserData::class.java)
+
+    override fun getUserData(): String? {
+        var userData = UserData(
+            emailDataStore.emailToken,
+            emailDataStore.emailUsername,
+            emailDataStore.nextAlias?.replace(DUCK_EMAIL_DOMAIN, ""),
+        )
+        return jsonAdapter.toJson(userData)
+    }
+
```

Comment:
There is not need to create private mutable properties here that also don't go through our dependency injection.
This model is rather simple so I'd rather use `JSONObject`
Also the signature doesn't need to be nullable.

```suggestion
    override fun getUserData(): String {
        return JSONObject().apply { 
            put("token", emailDataStore.emailToken)
            put("userName", emailDataStore.emailUsername)
            put("nextAlias", emailDataStore.nextAlias?.replace(DUCK_EMAIL_DOMAIN, ""))
        }.toString()
    }

```